app/apps/client/admin/appLogs.js

```diff
@@ -6,30 +6,32 @@ import { APIClient } from '../../../utils';
 import moment from 'moment';
 import hljs from 'highlight.js';
 
-Template.appLogs.onCreated(function() {
-	const instance = this;
-	this.id = new ReactiveVar(FlowRouter.getParam('appId'));
-	this.ready = new ReactiveVar(false);
-	this.hasError = new ReactiveVar(false);
-	this.theError = new ReactiveVar('');
-	this.app = new ReactiveVar({});
-	this.logs = new ReactiveVar([]);
-
-	const id = this.id.get();
-
+const laodData = (instance) => {
```

Comment:
Misspelling, should be `loadData` and not `laodData`

Reply:
Thanks